<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>429</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0">
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_Port"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_Open">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lab_open_state">
       <property name="text">
        <string>Closed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="edit_file">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Download File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_File">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Download">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_downloadFileInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_target">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>31</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>21</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Target:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>901</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ResetTarget</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RunAppliaction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PingTarget</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_percent">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>661</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Open_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Close">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_first_open">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Open First</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_first_close">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Close First</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_second_open">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_second_close">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_push_first">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>310</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Push First</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_second_push">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pull_first">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>310</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Pull First</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_second_pull">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>380</y>
      <width>170</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_door">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>129</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_cabinet1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>129</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_cabinet2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>129</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>450</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>450</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_LED">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>181</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Show</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
